src/managing-deis/platform-ssl.md

```diff
@@ -1,3 +1,58 @@
 # Platform SSL
 
-TODO (bacongobbler): rewrite for v2
+SSL/TLS is the standard security technology for establishing an encrypted link between a web server
+and a browser. This link ensures that all data passed between the web server and browsers remain
+private and integral.
+
+To enable SSL for your cluster and all apps running upon it, you can add an SSL key to your load
+balancer. You must either provide an SSL certificate that was registered with a CA or provide
+[your own self-signed SSL certificate](../reference-guide/creating_self_signed_ssl.md).
+
+## Installing SSL on a Load Balancer
+
+On most cloud-based load balancers, you can install a SSL certificate onto the load balancer
+itself. Any communication inbound to the cluster will be encrypted while the internal components
+of Deis will still communicate over HTTP.
+
+To enable SSL, you will need to open port 443 on the load balancer and forward it to port 80 on the
+routers. For AWS, you'll also need to add port 443 in the security group settings for your load
+balancer.
+
```

Comment:
And really, that's just _one_ way to do it.  The other way is to add healthchecks (which k8s would _not_ have done automatically) and then use the node port 80 on the back end.  This is actually slightly more efficient, network wise, because you avoid one hop through a kube-proxy by doing this since the _other_ configuration (with the high-numbered ports on the back end and no health check) could send the traffic to _any_ node and the node that receives the traffic might not actually be running a router.